test/runtests.jl

```diff
@@ -47,6 +47,22 @@ end
     @test result ≈ [-0.452598, -0.478878, -0.520674, -0.561428, -0.598661] atol=1e-5
 end
 
+@testset "Hybrid LDA and Hybrid GGA evaluate" begin
+    rho = [0.1, 0.2, 0.3, 0.4, 0.5]
+    sigma = [0.2, 0.3, 0.4, 0.5, 0.6]
+    result = zeros(Float64, 5)
+
+    # LDA
+    hyb_lda_xc = Functional(:hyb_lda_xc_bn05)
+    result = evaluate(hyb_lda_xc, rho=rho, derivatives=0).zk
+    @test result ≈ [-0.162837, -0.234719, -0.286980, -0.329442, -0.365816] atol=1e-5
```

Comment:
I trust you computed these values in some other way (i.e. not using Libxc.jl). Is this correct?

Reply:
Yep :-) I just did it with libxc in C with their example script on the website [here](https://tddft.org/programs/libxc/manual/libxc-5.1.x/), initializing [func_id to 588](https://gitlab.com/libxc/libxc/-/blob/master/src/funcs_key.c) for `hyb_lda_xc_bn05` and 496 for `hyb_gga_x_s12h`, and adding `case XC_FAMILY_HYB_LDA:` in the switch statement.  (I can send you the script if that's not clear).  The results are the same up to 1e-5.

Reply:
great. Then I'll merge!